__tests__/app.test.js

```diff
@@ -55,3 +55,34 @@ describe("GET /api", () => {
             });
     });
 });
+
+describe("GET /api/articles/:article_id", () => {
+    test("200: should respond with an object containing a single article corresponding to the provided article id", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body;
+
+                expect(article).toBeInstanceOf(Object);
+
+                expect(article).toHaveProperty("author"), expect.any(String);
+                expect(article).toHaveProperty("title"), expect.any(String);
+                expect(article).toHaveProperty("article_id"), expect.any(Number);
+                expect(article).toHaveProperty("body"), expect.any(String);
+                expect(article).toHaveProperty("topic"), expect.any(String);
+                expect(article).toHaveProperty("created_at"), expect.any(String);
+                expect(article).toHaveProperty("votes"), expect.any(Number);
+                expect(article).toHaveProperty("article_img_url"), expect.any(String);
+            });
+    });
```

Comment:
We could be more specific with the response here, as we are using a static test database we know what the data will look like as we have the data in the test data file. So we could hard code the expected response.